lib/Onyx.js

```diff
@@ -1,21 +1,22 @@
 /* eslint-disable no-continue */
-import {deepEqual} from 'fast-equals';
-import _ from 'underscore';
-import * as Logger from './Logger';
-import cache from './OnyxCache';
-import * as Str from './Str';
-import createDeferredTask from './createDeferredTask';
-import fastMerge from './fastMerge';
-import * as PerformanceUtils from './metrics/PerformanceUtils';
-import Storage from './storage';
+import { deepEqual } from "fast-equals";
```

Comment:
A lot of updates here that deviate from our style guide. Can we re-update to get these back to that standard?

Reply:
Ah, I just left prettier do it's thing, what is the configuration I should use?

Reply:
I cleaned all the formatting changes, should be a lot let noisy now